packages/indexer/src/sinks/drizzle/utils.ts

```diff
@@ -16,10 +15,7 @@ export const pgTable: PgTableFn = (name, columns, extraConfig?) => {
 };
 
 export const getDrizzleCursor = (
```

Comment:
> If you add a return type to this you should be able to avoid a lot of casting in the other functions right?

No, actually its inferring correct return type, but i must have added in some places while working and forgot to remove it.